SECURITY.md

```diff
@@ -2,10 +2,14 @@
 
 ## Supported Versions
 
-Distroless currently tracks debian 10 ([buster](https://packages.debian.org/buster/)) for packages. Distroless images are updated as debian packages are updated. No updates are created for earlier releases.
+Distroless currently tracks debian 11 ([buster](https://packages.debian.org/bullseye/) current) and debian 10 ([buster](https://packages.debian.org/bullseye/)) packages.
 
-Debian tracking information at https://github.com/GoogleContainerTools/distroless/blob/main/checksums.bzl is updated daily
+Debian tracking information at https://github.com/GoogleContainerTools/distroless/blob/main/debian_archives.bzl and generally is updated within 48 hours of debian publishing an update.
 
 ## Reporting a Vulnerability
 
-If a distroless image you are using contains a CVE or other vulnerability, please let the team know by creating an issue and pointing to the CVE or vulnerability disclosure. Please include a link to the package in question from the debian package index for debian 10/buster (https://packages.debian.org/index)
+If a distroless image you are using contains a CVE or other vulnerability:
```

Comment:
It would be useful to turn this into a numbered list so that it's clear they are steps.

Reply:
done